app/src/main/res/layout/activity_git_clone.xml

```diff
@@ -19,30 +19,76 @@
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintTop_toTopOf="parent" />
 
+    <View
+        android:id="@+id/spacer"
+        android:layout_width="match_parent"
+        android:layout_height="1dp"
+        android:layout_marginTop="8dp"
+        android:layout_marginBottom="8dp"
+        android:background="?android:attr/textColorPrimary"
```

Comment:
This line is causing the crash. It goes away when I change it to backgroundTint. Maybe we should just get rid of the `spacer` since it doesn't seem to be needed here?

Reply:
Yeah the spacer is unnecessary. Thanks for triaging.